---
title: MSSQLSERVER_8630 | Microsoft Docs
ms.custom: ''
ms.date: 04/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: supportability
ms.topic: language-reference
helpviewer_keywords:
- 8630 (Database Engine error)
ms.assetid: 45a9d084-b5ad-435b-ac19-5f4352515817
author: MashaMSFT
ms.author: mathoma
manager: craigg
ms.openlocfilehash: f81fb0efe4fd44feb17e30326e206e49e48168f5
ms.sourcegitcommit: 3026c22b7fba19059a769ea5f367c4f51efaf286
ms.translationtype: HT
ms.contentlocale: ja-JP
ms.lasthandoff: 06/15/2019
ms.locfileid: "62656349"
---
# <a name="mssqlserver8630"></a>MSSQLSERVER_8630
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
  
## <a name="details"></a>詳細  
  
|||  
|-|-|  
|製品名|SQL Server|  
|イベント ID|8630|  
|イベント ソース|MSSQLSERVER|  
|コンポーネント|SQLEngine|  
|シンボル名|EXECUTION_ERR|  
|メッセージ テキスト|内部クエリ プロセッサ エラー:クエリ プロセッサ実行中に予期しないエラーが発生しました。|  
  
## <a name="explanation"></a>説明  
クエリ処理中に内部エラーが発生しました。  
  
## <a name="user-action"></a>ユーザーの操作  
主要なサービス プロバイダーに連絡し、 このエラー メッセージの情報を提供します。  
  
